action.yml

```diff
@@ -19,6 +19,13 @@ inputs:
     description: Comma separated list of file extentions to lint.
     required: false
     default: php,module,inc,install,test
+  phpcs-ignore:
+    description: Comma separated list of files to ignore.
+    required: false
+    default: .md
```

Comment:
Was spec'd a `*.md` originally... is the `*` relevant?

Reply:
This didn't seem relevant or affect the testing whether it was there or not, but considering it was originally there and worked properly, including it back.